addons/binding/org.openhab.binding.globalcache/src/main/java/org/openhab/binding/globalcache/handler/GlobalCacheHandler.java

```diff
@@ -264,7 +272,7 @@ private String lookupCode(Command command, ChannelUID channelUID) {
                 code = convertHexToGC(code);
                 logger.debug("Converted hex code is: {}", code);
             } catch (HexCodeConversionException e) {
-                logger.warn("Failed to convert hex code to GC format");
+                logger.info("Failed to convert hex code to GC format");
```

Comment:
Yes, messing with IR codes can be... well, I guess a bit messy.  :-)

I suppose I could update the thing status to show it online, but with a detailed status message (I think that's possible?).  But that didn't seem any better solution than logging an info level message.